spyder/plugins/editor/plugin.py

```diff
@@ -774,6 +774,14 @@ def get_plugin_actions(self):
                                add_shortcut_to_tip=True)
 
         # --- Edit Toolbar ---
+        create_new_cell = create_action(self, _("Create new Cell at "
+                                                "current line"),
+                                        icon=ima.icon('run_selection'),
```

Comment:
Lets change the icon for now to the `cell` icon (![image](https://user-images.githubusercontent.com/16781833/218288263-fc225c6e-1323-4de3-9fec-20399eb07f3f.png)):
```suggestion
                                        icon=ima.icon('cell'),
```